www/src/pages/blog/astro-018.astro

```diff
@@ -0,0 +1,112 @@
+    <style>
+      img {
+        max-width: 100%;
+      }
+    </style>
+  </head>
+
+  <body>
+    <BlogHeader />
+    <BlogPost title={title} author={author} heroImage={heroImage} publishDate={publishDate}>
+        <Markdown>
+          After the success of Astro's launch we spent some time fixing bugs and taking user feedback on what to tackle next. Now some of the features that people requested are starting to get released.
+          
+          Astro 0.18 is the culmination and is the biggest release to date. This post highlights only some of what it brings.
+
+          ![Code example that shows off using the new client:media hydrator.](/assets/blog/astro-018/responsive-hydration.png)
+          <!-- Saved from https://carbon.now.sh/16xchqPVdt5IEAY3Czg3 -->
+
+          ## Responsive component hydration
+
+          Astro uses [partial hydration](https://docs.astro.build/core-concepts/component-hydration), a technique to only include the client-side JavaScript for the parts of a page that are actually interactive. With Astro there were previously 3 strategies for partial hydration: hydrating on [page load](https://docs.astro.build/core-concepts/component-hydration#mycomponent-clientload-), when the CPU is [idle](https://docs.astro.build/core-concepts/component-hydration#mycomponent-clientidle-), or when the element is scrolled into [view](https://docs.astro.build/core-concepts/component-hydration#mycomponent-clientvisible-).
+
+          When developing responsive sites you often have components that are only used on certain device sizes, like a mobile sidebar menu. Many sites will load their mobile-only code on all devices, wasting performance for users who don't ever see this UI.
+
+          To solve that problem we've now added a 4th way to hydrate, using a **[CSS media query](https://docs.astro.build/core-concepts/component-hydration#mycomponent-clientmediaquery-)**. Add the `client:media` directive to a component and it will only hydrate once the media condition is met.
+
+          This is all part of Astro's commitment to providing tools to minimize the amount of JavaScript you load.
+
+          ![Using multiple slots to provide the head and body parts of a Layout component.](/assets/blog/astro-018/named-slots.png)
+          <!-- https://carbon.now.sh/9UwJkMCezRIOhzac5VVp -->
+
+          ## Named slots
+
+          [Named slots](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots) are a standardized feature in shadow DOM and used in frameworks like [Vue](https://vuejs.org/v2/guide/components-slots.html#Named-Slots). In this release, Astro adds [named slots to Astro components](https://docs.astro.build/core-concepts/astro-components#slots), pages, and layouts.
+
+          Named slots are particularly useful for layout components. Layout components are what we call components that provide common HTML for a page, such as commonly added CSS in the &lt;head&gt; and the page's skeleton content.
```

Comment:
don't think you need this paragraph, since I believe the general concept of a layout is generally understood. 

you can just link the first reference to `Layout` in the next paragraph to our docs.